src/CheckBoxGroup/CheckBoxGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+
+import { CheckBox } from './CheckBox'
+
+type Item = {
+  id: number
+  label: string
+  value: string
+  checked: boolean
+}
+
+type Props = {
+  list: Item[]
+  onToggle: (id: number) => void
+}
+
+export const CheckBoxGroup: FC<Props> = ({ list, onToggle }) => {
+  return (
```

Comment:
Can you remove the return statement and the `{ }` and replace it with `=> ( ...<div's>...)`